ctapipe_io_nectarcam/__init__.py

```diff
@@ -30,6 +30,11 @@
 from astropy.io import fits
 from astropy.time import Time
 from .containers import NectarCAMDataContainer
+from .anyarray_dtypes import (
+    CDTS_AFTER_37201_DTYPE,
```

Comment:
The 37201 is an LST run number, probably doesn't make too much sense to keep it here like this for nectarcam

Reply:
Does it refer to an LST run number? I thought it was a CDTS subversion revision. I just tried to check the history of svn+ssh://svn.in2p3.fr/cta/ACTL/ExternalDevicesCommunication and cannot reach a definite conclusion but the time fits more or less with the introduction of the new format (Autumn 2019).

Reply:
We check here:

https://github.com/cta-observatory/ctapipe_io_lst/blob/afac9ce3889d5e4c66042d08ba1b9135f3a4cdae/ctapipe_io_lst/__init__.py#L531

instead of checking the byte size of the data